src/server/pegasus_write_service.cpp

```diff
@@ -297,16 +297,18 @@ int pegasus_write_service::duplicate(int64_t decree,
         resp.__set_error(_impl->multi_remove(ctx.decree, rpc.request(), rpc.response()));
         return resp.error;
     }
+    put_rpc put;
```

Comment:
I prefer to name them as put_rpc_ and remove_rpc_.

Reply:
We disallow using prefixed/suffixed underscore to name a local variable. Because they are used to distinguish class members from local variables. This convention is not only suitable for C++, but also other languages.

Reply:
Well, I have reset it to `put_rpc`